engine/security/rootless.md

```diff
@@ -299,22 +307,21 @@ Or add `net.ipv4.ip_unprivileged_port_start=0` to `/etc/sysctl.conf` (or
 In Docker 19.03, rootless mode ignores cgroup-related `docker run` flags such as
 `--cpus`, `--memory`, --pids-limit`.
 
-However, traditional `ulimit` and [`cpulimit`](https://github.com/opsengine/cpulimit)
-can be still used, though they work in process-granularity rather than in container-granularity,
-and can be arbitrary disabled by the container process.
+However, you can still use the traditional `ulimit` and [`cpulimit`](https://github.com/opsengine/cpulimit), though they work in process-granularity rather than in container-granularity, and can be arbitrarily disabled by the container process.
```

Comment:
```suggestion
However, you can still use the traditional `ulimit` and [`cpulimit`](https://github.com/opsengine/cpulimit),
though they work in process-granularity rather than in container-granularity,
and can be arbitrarily disabled by the container process.
```